### PR TITLE
Use selectedValue instead of valueProperty.

### PR DESCRIPTION
**Pull request by abuiles**

### CODE REVIEW

**Comment by abuiles** [comment]
Such breaking change :P

File: README.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ First add a custom component that extends AutoComplete. In this component you ne
 ```
 1) suggestions:    this computed will determine how the list of options is filtered as the user enters text
 2) optionsToMatch: this computed will determine if the value entered is valid (when the user omits to click/enter/tab the selection)
-3) valueProperty:  this string should be the value property for the options passed in (think selectbox value/label)
+3) optionLabelPath:  this string should be the value property for the options passed in (think selectbox value/label)
```
Comment (abuiles) [inline README.md:26]: Actually, this should be `optionValuePath` instead.

File: tests/dummy/app/components/my-auto-complete.js [hunk 2]
```diff
@@ -16,5 +16,5 @@ export default AutoComplete.extend({
           caseInsensitiveOptions.push(value.toLowerCase());
       });
       return caseInsensitiveOptions;
-  }.property("options.@each")
+  }.property("options.[]")
```
Comment (toranb) [inline tests/dummy/app/components/my-auto-complete.js:19]: I see this change from options.@each to options.[] but I don't know what one offers over the other. You have a blog post I could read/source link to ember that explains this as a win/win ?

Comment (abuiles) [inline tests/dummy/app/components/my-auto-complete.js:19]: https://github.com/abuiles/ember-cli-auto-complete/commit/2af15dbcfe51817e2c9ef1b3c869abf35848fb78

**Comment by abuiles** [comment]
@toranb please don't merge this yet, let me do a bunch of things on top of this branch and then we can merge in.

File: tests/dummy/app/components/my-auto-complete.js (hunk 2 above)
Comment (toranb) [inline tests/dummy/app/components/my-auto-complete.js:19]: haha and if only I had read your comment block for that specific commit :)

**Comment by toranb** [comment]
Thanks for the heads up - I'll track along side you on this one

**Comment by toranb** [comment]
Is this PR still needed/ desirable ? If yes, what can I do to help? If no, can we close it out?

**Comment by abuiles** [comment]
@toranb sorry man, I lost track of this, I think extending for this is better, it worked great for me, I'll try to open source a select-component I built out of this.

**Comment by abuiles** [comment]
@toranb  put it in a gist in the meantime https://gist.github.com/abuiles/3353197ac9e1881bc5af

**Comment by abuiles** [comment]
Also included the template

**Pull request closed without merging**